File: daySix/javaScript/MoveAllZerosInTheArray.js
function MoveAllZeroIn(numbers){
    
    for(let index = 0; index < numbers.length; index++){
        for(let secondIndex = index + 1; secondIndex < numbers.length; secondIndex++){
            if(numbers[index] < numbers[secondIndex]){
                let temp = numbers[index];
                numbers[index] = numbers[secondIndex];
                numbers[secondIndex] = temp;
            }
        }
    }
    return numbers;
}
let numbers = [5,0,2,0,3,0];
console.log(MoveAllZeroIn(numbers))

File: daySix/java/QuestionThree/MoveAllZerosInTheArray.java
public class MoveAllZerosInTheArray {
    public static int[] moveAllZerosIn(int[] numbers){
        int temp = 0;
        for(int index = 0; index < numbers.length; index++){
            for(int secondIndex = index + 1; secondIndex < numbers.length; secondIndex++){
                if(numbers[index] < numbers[secondIndex]){
                    temp = numbers[index];
                    numbers[index] = numbers[secondIndex];
                    numbers[secondIndex] = temp;
                }
            }
        }
        return numbers;
    }
}
